core/src/main/java/tc/oc/pgm/ffa/FreeForAllModule.java

```diff
@@ -86,6 +86,15 @@ public FreeForAllModule parse(MapFactory factory, Logger logger, Document doc)
           colors = XMLUtils.parseBoolean(Node.fromAttr(elPlayers, "colors"), colors);
         }
 
+        if (minPlayers > maxPlayers) {
+          if (elPlayers != null)
+            throw new InvalidXMLException("\"max\" must be greater than \"min\"", elPlayers);
+          else {
+            if (maxPlayers > 1) minPlayers = maxPlayers / 2;
```

Comment:
I belive it makes more sense to force minPlayers to maxPlayers (bukkit max) if it goes over the limit. That's the biggest minimum that will be able to be met

Reply:
Same, PGM should not care about `Bukkit#getMaxPlayers` anywhere.